multi_partner_learning.py

```diff
@@ -137,7 +142,13 @@ def compute_test_score(self):
         # Initialize variables
         model_to_evaluate, sequentially_trained_model = None, None
         if self.learning_approach in ['seq-pure', 'seq-with-final-agg']:
-            sequentially_trained_model = self.generate_new_model()
+            if self.use_saved_weights:
+                logger.info(f"(seq) Init models with previous coalition model for each partner")
+            else:
+                logger.info(f"(seq) Init new models for each partner")
```

Comment:
Why do we log that in the `if self.learning_approach in ['seq-pure', 'seq-with-final-agg'] `?

Reply:
Ok understood